gammapy/modeling/models/spectral.py

```diff
@@ -1422,13 +1422,23 @@ class AbsorbedSpectralModel(SpectralModel):
         parameter value for absorption model
     parameter_name : str, optional
         parameter name
+    alpha_norm: float
+        Norm of the EBL model, by default 1.
+    alpha_norm_frozen: bool
```

Comment:
@JouvinLea - Please remove this `alpha_norm_frozen` option.
Just put `frozen=True` in the `Parameter` call in `__init__` below.
If someone wants to change this, they can change to `model.alpha_norm.frozen = False` after `__init__`.
This is what we do for all other cases where parameters are usually frozen, e.g. the reference energies in spectral models.
We will improve the model init methods, but manually adding options like `alpha_norm_frozen` is not the way to go.